assets/src/apps/authoring/components/AdaptivityEditor/InitStateEditor.tsx

```diff
@@ -142,8 +157,8 @@ interface TypeOption {
   value: number;
 }
 const typeOptions: TypeOption[] = [
-  { key: 'number', text: 'Number', value: 1 },
   { key: 'string', text: 'String', value: 2 },
+  { key: 'number', text: 'Number', value: 1 },
```

Comment:
should probably factor these options out to a shared file

Reply:
fixed